src/components/ListItem/ListItem.types.ts

```diff
@@ -13,6 +13,11 @@ export type ListItemProps = {
      */
     label: React.ReactNode;
 
+    /**
+     * ListItem description
```

Comment:
Please, use the same description for the same props
```suggestion
     * Optional description to display to describe the item
```

Reply:
Done